lookout/style/typos/cmdline_tools.py

```diff
@@ -60,11 +60,13 @@ def cli_get_datasets(data_path: str, config: Mapping[str, Any]) -> None:
 
 
 def cli_train_corrector(train: str, test: str, vocabulary_path: str,
-                        frequencies_path: str, fasttext_path: str, corrector_path: str) -> None:
+                        frequencies_path: str, fasttext_path: str, corrector_path: str,
+                        config: Mapping[str, Any]) -> None:
     """Entry point for `train_and_evaluate`."""
     train = pandas.read_csv(train, index_col=0, keep_default_na=False)
     test = pandas.read_csv(test, index_col=0, keep_default_na=False)
-    model = train_and_evaluate(train, test, vocabulary_path, frequencies_path, fasttext_path)
+    model = train_and_evaluate(train, test, vocabulary_path, frequencies_path, fasttext_path,
+                               config.get("generation", {}), config.get("ranking", {}))
```

Comment:
let's try to avoid as many `config.get("something", {})`  as possible. You can merge the config with a `DEFAULT_CONFIG` via `merge_dicts` function.

I think with this change many configs in `__init__` methods become non-optional which is good.

Reply:
It's the only place i do that, here it's unavoidable: we get the config with possible `"ranking" and "generation"` fields, but the `train_and_evaluate` function expects them separately.

Reply:
```python
config = merge_dicts(DEFAULT_CONFIG, config)
model = train_and_evaluate(train, test, vocabulary_path, frequencies_path, fasttext_path,
                               config["generation"], config["ranking"]))
```
Can be a solution?

Reply:
No, too much excessive information will be added that way

Reply:
BTW about merge_configs - what would really improve the situation is to accept also None dict and treat is as empty one.

Reply:
The colution here would be for example to do smth like that `"corrector": {"generation": {}, "ranking"}}`(and pass this `config["corrector"]` to the function). This was actually my first idea, but i decided to change it to the present one to avoid the growth of the nesting (`DEFAULT_CORRECTOR_CONFIG["corrector"]["generation"]).

Reply:
ok, let's keep it